tests/cephfs/clients/mds_conf_modifying.py

```diff
@@ -0,0 +1,187 @@
+            ("mds_dirstat_min_interval", "2.000000"),
+            ("mds_scatter_nudge_interval", "6.000000"),
+            ("mds_client_prealloc_inos", "1001"),
+            ("mds_early_reply", "false"),
+            ("mds_default_dir_hash", "3"),
+            ("mds_log_skip_corrupt_events", "true"),
+            ("mds_log_max_events", "0"),
+            ("mds_log_max_segments", "256"),
+            ("mds_bal_sample_interval", "4.000000"),
+            ("mds_bal_replicate_threshold", "8001.000000"),
+            ("mds_bal_unreplicate_threshold", "0.100000"),
+            ("mds_bal_split_size", "10001"),
+            ("mds_bal_split_rd", "25001.000000"),
+            ("mds_bal_split_wr", "10001.000000"),
+            ("mds_bal_split_bits", "6"),
+            ("mds_bal_merge_size", "51"),
+            ("mds_bal_interval", "11"),
+            ("mds_bal_fragment_interval", "6"),
+            ("mds_bal_fragment_fast_factor", "1.700000"),
+            ("mds_bal_fragment_size_max", "100001"),
+            ("mds_bal_idle_threshold", "0.100000"),
+            ("mds_bal_max", "1"),
+            ("mds_bal_max_until", "1"),
+            ("mds_bal_mode", "1"),
+            ("mds_bal_min_rebalance", "0.200000"),
+            ("mds_bal_min_start", "0.300000"),
+            ("mds_bal_need_min", "0.900000"),
+            ("mds_bal_need_max", "1.500000"),
+            ("mds_bal_midchunk", "0.500000"),
+            ("mds_bal_minchunk", "0.005000"),
+            ("mds_replay_interval", "1.500000"),
+            ("mds_shutdown_check", "1"),
+            ("mds_thrash_exports", "1"),
+            ("mds_thrash_fragments", "1"),
+            ("mds_dump_cache_on_map", "true"),
+            ("mds_dump_cache_after_rejoin", "true"),
+            ("mds_verify_scatter", "true"),
+            ("mds_debug_scatterstat", "true"),
+            ("mds_debug_frag", "true"),
+            ("mds_debug_auth_pins", "true"),
+            ("mds_debug_subtrees", "true"),
+            ("mds_kill_mdstable_at", "1"),
+            ("mds_kill_export_at", "1"),
+            ("mds_kill_import_at", "1"),
+            ("mds_kill_link_at", "1"),
+            ("mds_kill_rename_at", "1"),
+            ("mds_inject_skip_replaying_inotable", "true"),
+            ("mds_kill_skip_replaying_inotable", "true"),
+            ("mds_wipe_sessions", "true"),
+            ("mds_wipe_ino_prealloc", "true"),
+            ("mds_skip_ino", "1"),
+            ("mds_min_caps_per_client", "101"),
+        ]
+
+        for target in conf_target:
+            client_conf = target[0]
+            value = target[1]
+            client1.exec_command(sudo=True, cmd=f"{conf_set} {client_conf} {value}")
+            output = client1.exec_command(
+                sudo=True, cmd=f"ceph config get mds {client_conf}"
+            )
+            if output[0].rstrip() != target[1]:
+                raise CommandFailed(f"Failed to set {client_conf} to {value}")
+        log.info("Successfully set all the client config values")
```

Comment:
post modifying mds configuration values, test case mentions to run IO and validate no crashes or corruption. Is this not planned to be covered?

Reply:
will be covered in the fixed commit.

Reply:
updated